.test-env

```diff
@@ -1,6 +1,6 @@
 # Configs for testing repo download:
```

Comment:
@algochoi When you return to the PR - Leaving a visual reminder to add a CHANGELOG entry.

Reply:
Thanks - added in [8f2d18b](https://github.com/algorand/go-algorand-sdk/pull/451/commits/8f2d18b76c96dac97a4275815047d2211c0a21c2)